SocialCareCaseViewerApi/V1/Helpers/Paginated.cs

```diff
@@ -6,5 +6,6 @@ public class Paginated<T>
     {
         public List<T> Items { get; set; }
         public long Count { get; set; }
+        public long DeletedItemsCount { get; set; }
```

Comment:
Instead of modifying this generic class could we extend it?

I'm only asking this because I know this class will be used elsewhere but in all cases we won't have DeletedItems

Reply:
That's fine, I have added a new class that inherits from this one. Not sure about the name, not very imaginative. I tried to keep that quite generic as well, but happy to rename.

```
public class PaginatedExtended<T> : Paginated<T>
{
	public long DeletedItemsCount { get; set; }
} 
```

this inheritance makes the json properties to appear in funny order i.e. the extended class' properties come first. I don't think that matters as far as code handling is concerned.